client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -178,15 +207,15 @@ class EditContactInfoFormCard extends React.Component {
 				'We’ll email you at {{strong}}%(oldEmail)s{{/strong}} and {{strong}}%(newEmail)s{{/strong}} ' +
 					'with a link to confirm the new details. The change won’t go live until we receive confirmation from both emails.',
 				{
-					args: { oldEmail: currentEmail, newEmail: newContactDetails.email },
+					args: { oldEmail: email, newEmail: newContactDetails.email },
 					components: { strong },
 				}
 			);
 		} else {
 			text = translate(
-				'We’ll email you at {{strong}}%(currentEmail)s{{/strong}} with a link to confirm the new details. ' +
+				'We’ll email you at {{strong}}%(email)s{{/strong}} with a link to confirm the new details. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'We’ll email you at {{strong}}%(currentEmail)s{{/strong}} with a link to confirm the new details. The change won't go live until we receive confirmation from this email.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).